usecases/stable/replay.recording/Dockerfile

```diff
@@ -17,5 +17,5 @@
 
 # Date: 2016-12-02
 
-FROM seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete:v0.7.1
+FROM seresearch/opendlv-core-on-opendavinci-ubuntu-16.04-complete:latest
```

Comment:
What will be the next version number? I will update it for all relevant stable use cases accordingly